zt_frontend/src/App.vue

```diff
@@ -55,11 +118,96 @@
         @createCell="createCodeCell"
        />
     </v-main>
+    <v-footer 
+      app
+      class="footer bg-bluegrey-darken-4 text-bluegrey"
+    >
+      <div class="footer__left-container">
+        <v-icon
+          class="footer__code-version"
+          icon="$cubic" 
+        />
+        <span>Python 3.9</span>
+        <v-icon class="footer__dot-divider" icon="$dot"/>
+        <span>Zero-True v1.1</span>
+        <v-icon class="footer__dot-divider" icon="$dot"/>
+        <span>{{ cellLength }} cells</span>
+      </div> 
+      <div class="footer__right-container">
+        <div>
+          <v-progress-circular
```

Comment:
For the timer and the queue these should only be displayed when isCodeRunning is true

Reply:
Can we also add a stop button similar to what was in the top bar previously? When clicked it should call the same method as before